server/ServerFSM.cpp

```diff
@@ -622,6 +622,59 @@ sc::result Idle::react(const Error& msg) {
 }
 
 sc::result Idle::react(const Hostless&) {
+    std::string autostart_load_filename = GetOptionsDB().Get<std::string>("load");
```

Comment:
so the `--load "filename"` option only works in hostless mode on the server? Should indicate this in the help string.

Reply:
Added both.